parser/src/util.ts

```diff
@@ -74,11 +74,14 @@ export async function deflateGitObject(repo: string, hash: string) {
 }
 
 export async function writeRepoToFile(
-  commitObject: GitCommitObject,
+  commitObject: HydratedGitCommitObject,
+  repoName: string,
+  branchName: string,
   repoDir: string,
   outFileName: string
 ) {
-  const data = JSON.stringify(commitObject, null, 2)
```

Comment:
This is not the responsibility of `writeRepoToFile`.
This should be done elsewhere